src/urllib3/poolmanager.py

```diff
@@ -62,6 +63,11 @@
     "key_server_hostname",  # str
 )
 
+# Python 3.7 introduces the parameter "blocksize" to
+# http.client.HTTPConnection's and http.client.HTTPSConnection's initializers
+if sys.version_info >= (3, 7):
+    _key_fields += ("key_blocksize",)  # int
```

Comment:
Within the `_default_key_normalizer` we should set the default to be `16384` instead of `None` so that users get a more desirable default.

Can we also get a test case that the normalizer sets `blocksize` properly given inputs like no `blocksize` set, blocksize=8192, blocksize=16384

Reply:
I'm passing `blocksize = 16384` to the pool class in _new_pool if it is not set in the `request_context`, but I'm afraid this may be too intrusive. What do you think?